src/java.desktop/share/classes/sun/java2d/cmm/lcms/LCMSTransform.java

```diff
@@ -66,13 +66,6 @@
 
     private Object disposerReferent = new Object();
 
-    /* the class initializer */
-    static {
-        if (ProfileDeferralMgr.deferring) {
-            ProfileDeferralMgr.activateProfiles();
```

Comment:
We have to activate it here because later in this class we call:
76        lcmsProfiles[0] = LCMS.getProfileID(profile);
And that native method expects the activated profile, after the fix, we will activate it in that native method.